docs/examples/base/strong.html

```diff
@@ -0,0 +1,7 @@
+---
+layout: examples
+title: Strong text
+category: _base
+---
+
+<strong>This is some &lt;strong&gt; text.</strong>
```

Comment:
I think it would be a better example if the strong post of a sentence. Can you try make the sentence more real. Like: The pastebin is **blocked**